build.gradle.kts

```diff
@@ -0,0 +1,33 @@
+plugins {
+    id("org.springframework.boot") version "2.5.2"
+    id("io.spring.dependency-management") version "1.0.11.RELEASE"
```

Comment:
Cares about using proper versions in the project. Allows skipping defining version in the dependencies block.